include/aos/iam/certhandler.hpp

```diff
@@ -18,37 +18,12 @@
 #include "aos/common/tools/thread.hpp"
 #include "aos/common/tools/utils.hpp"
 #include "aos/common/types.hpp"
+#include "aos/iam/certmodule.hpp"
 
 namespace aos {
 namespace iam {
 namespace certhandler {
 
-/**
```

Comment:
1. In general, no need to move CertModuleItf into separate file. Usually, we define interfaces in the same header where the client of these interfaces is defined. Exceptions are generic interfaces used by many clients. In this particular case `CertModuleItf` could be placed in separate file but I would like to have clean strict rules. So, keep it in the `certhandler.hpp` now.
2. These changes are not related to this commit.